scripts/rollup/build.js

```diff
@@ -185,7 +186,8 @@ function getRollupOutputOptions(
     format,
     globals,
     freeze: !isProduction,
-    interop: false,
+    interop: 'default',
```

Comment:
The interop configuration uses the default values to minimize the size increase of the production bundles and pass the ci, and to be honest I'm not very confident about this change.

Reply:
From looking at the generated output files, I think this looks mostly right.

There's still a bit of extra code here (you can see the diffs in this comment https://github.com/facebook/react/pull/26078#issuecomment-1408052671 )

Example larger build where the change is visible: https://react-builds.vercel.app/commits/cc7dbd62a503bf502eb911b82a28926cf4d177e9/files/oss-stable/react-debug-tools/cjs/react-debug-tools.development.js?compare=4a4ef2706cfb51db96d48fe20dadcb4fb8e3cb17

Reply:
Pretty sure this should be `'esModule`' instead of `'default'`